### DISCUSSION TITLE
#62 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Thegaram** [tog-ETH-er/togETHer#47] [General]
Agenda:

- Community Updates
- 下一任轮值主席提名
  > 轮值主席从正式成员中产生。前期，综合成员主观意愿、积极程度、组织能力等因素，『原语里弄』作为首任轮值主席。任何正式成员都可报名轮值主席。常规流程下，下一任轮值主席由当任轮值主席发起提名。[[source](https://docs.google.com/document/d/1RD491LNYSRvfe-CAU2wdWVqdT_Ru8gXJJj8bwKMAmpE/edit#)]
  - 欢迎各位社区来申请
- Fireside Chat
  - GCC参与方式
  - EDCON
  - Devcon location [vote](https://forum.devcon.org/c/devcon-7-location-suggestions/14?order=posts)
  - 如何吸引新的人加入社区

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/wba-mxnt-bsu

**Comment by Thegaram** [comment]
### CryptoTechNight

规划下一期CTN活动。

**Comment by Stamp9** [comment]
### 原语里弄

研究项目进行中。

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

5个PR #956 ～ #961

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了1场分享会

 
- 《密码学系列课程6:  GG18门限签名》by lynndell
  - Youtube:
    - p1 https://youtu.be/yN3OvR82DHc
    - p2 https://youtu.be/Jk3SK1IJc5Y

  - Bilibili:
    - p1 https://www.bilibili.com/video/BV1ds4y1c7Xa
    - p2 https://www.bilibili.com/video/BV1Ps4y1R7BD/

**Comment by ethz42** [comment]
PlanckerDAO
--

### 运营小组 @Lauran

- 4月各类活动的物料设计bounty结算完毕
- Notion改版
    - 找了706负责notion的小伙伴，5.1号后每天投入4个小时，运营小组会定期组织反馈
- 黑山社交联系表格方案 + Discord 信息整理
- 社区品牌媒体工作方案ing
    - 先出一个方案再找到合适的媒体专业人士参与进来

### 公共池 public pool

GCC

1. twitter space本周要启动，邀请参与的项目方
2. 在黑山募资需要的GCC的deck

**DAOStar小组**

1.升级为Daostar CN tg群

2.gcc小组协助daostar op申请了gcc gitcoin round

**英语共学**

**小V共读**

**黑山**

**EIPs FUN**

1. 官网上线 [https://eipsfun-preview.vercel.app/#](https://eipsfun-preview.vercel.app/#)
2. 产品deck

[https://docsend.com/view/xjfw7uhykbjkhw9a](https://docsend.com/view/xjfw7uhykbjkhw9a)

1. grant pool推进中

### 项目组

**Bundler**

**LSD稳定币**

与Pablo、焦哥、圣洁、金伯上周继续举行同步分享会，Kenway分享了关于LSD稳定币的proposal草稿；Pablo分享了最近研究的Frax的稳定币项目跟Vitalik的一篇关于稳定币的实验文章；一起讨论了稳定币的一些特点，后面需要大家分头去研究市面上存在的稳定币项目。

**Remix China**

1. 给 Remix 团队发了翻译计划第一笔预算的 invoice，目前还没收到回复
2. crowdin 上 Remix IDE 使用文档的翻译完成度已达 100% ，接下来 mocca 会把翻译完的文档部署到国内镜像站
3. 跟星火链网的人约了线上会议讨论 Remix 插件的事儿，他们很感兴趣，打算委托 Remix China 来开发插件。接下来我会整理一份需求文档，然后跟星火链网那边过下需求。
4. 发现了 Remix IDE 原有的 git 功能有缺陷，后面会对 git 功能做优化。 @0xRory 在看了。以后类似这种我们国内社区发现的功能缺陷或者 bug ，都会纳入我们的开发计划内。
### 运营小组

- Plancker 品牌媒体运营方案 [[Plancker 社区品牌与媒体工作方案](https://www.notion.so/Plancker-990ac39868dd4df4a7e5fcaa766bce2c)](https://www.notion.so/Plancker-990ac39868dd4df4a7e5fcaa766bce2c)
    - 运营会议上讨论了修改意见，周三前会修改完成
- [[社区资源情况整理](https://www.notion.so/6f5c8ef948d14f71b172c66e2da0c09b)](https://www.notion.so/6f5c8ef948d14f71b172c66e2da0c09b)
    - 接下来会尝试优化&明确的：
        - 社区整体来说希望系统性输出的内容/策展内容
        - 阶段性目标的确定（在品牌媒体方案中有提到）

### 公共层

**DAOStar小组**

这周会参与一个针对亚洲区Community Roundtable

**英语共学**

**Plancker Ethereum Talk**

已经与分享者们进行沟通，于北京时间4月8日晚8点进行第一次试运行

材料见如下链接

[[会议第一次试运行](https://www.notion.so/bb4e3206aad44750a3d464e74c705129)](https://www.notion.so/bb4e3206aad44750a3d464e74c705129) 

**小V共读**

1.策划5-13的分享

2.GCC和一些社区事务支持（zuzalu）

3.Arvin开始负责567的主理工作

个人：大部分时间投入以太坊生态和技术研究，部分时间服务公共层事务；摒弃绝大部分社交。

以太坊研究会定期分享输出，也期待酝酿下一个好玩的公共物品。

### 项目组

---

**EIPs.FUN**

[[EIPs.fund](https://www.notion.so/EIPs-fund-ae124a2c15e04eddacf69fadeaafdd53)](https://www.notion.so/EIPs-fund-ae124a2c15e04eddacf69fadeaafdd53) 

[[Proposal: Grants for EIPs Fun Project](https://forum.plancker.org/t/proposal-grants-for-eips-fun-project/93/2)](https://forum.plancker.org/t/proposal-grants-for-eips-fun-project/93/2)

**Bundler**

- 暂无进展

**Restake-based RAI**
继续探索如何利用Restake机制来实现RAI的抵押流程，研究了一下EigenLayer，写了个总结notion，后面会分享一下；

金伯：最近在看tokenomics，在分析RAI DAI的区别（比起DAI RAI更像是一个中央银行），学习代币稳定机制（目前主要是提供套利机会，会在技术研讨会上进行分享）后续计划会找个时间了解一下RAI DAI的交互流程，以及研究RAI控制器部分的代码

**Remix China**

- 翻译计划
    - invoice 有点小问题，修改后又发过去了，接下来就等 Remix 团队那边打款了
    - Remix 团队的 Rob 说要进行 regular calls，定期同步下这边的进度。我建议让他直接来参加 Remix China 双周会，双周会时间可能需要调整。后面得克服下英语障碍，有点脑壳疼。
- 尝试写了一篇文章，但感觉写的不好。总是想在一篇文章中把所有想说的点都说清楚，搞得自己压力很大，写起来很费劲。我打算转变下思路，把一篇拆成很多小篇来写。每篇文章500到800字，每次只阐述一个主题。后续可能每1到2周产出一篇。
- 把星火链网Remix插件的的需求整理成了文档，接下来会再约一下星火链网的人，过一下需求。

**许愿池**

- 进行了基础的架构拆分
- 上周进行了会议同步
    - [[5月5日许愿池建设讨会议](https://www.notion.so/5-5-dc6ff48326e34e838478adbfcfcb68de)](https://www.notion.so/5-5-dc6ff48326e34e838478adbfcfcb68de)
- 接下来要做 demo 的设计和进一步开发
    - 希望能够和 GCC 做结合，通过许愿池来为 GCC 进行募资，但是需要评估各自的进度，不强求结合。
- 5月19日 要在 edcon 的 superdemo 进行展示

******GCC******

- 可能要拓展名称
    
    [[Universal Digital Commons of GCC社区简介](https://www.notion.so/Universal-Digital-Commons-of-GCC-27e3729123c846f8b756dbadaa1b5c84)](https://www.notion.so/Universal-Digital-Commons-of-GCC-27e3729123c846f8b756dbadaa1b5c84) 
    
- zuzalu 在地会有建设者加入
- 5月22号计划举行讨论
    - [[中文区区块链的现状、以及其他语区的探讨](https://www.notion.so/7bbb6ac8fd7f4c6eb720a63f3ff8af9d)](https://www.notion.so/7bbb6ac8fd7f4c6eb720a63f3ff8af9d)
- 许愿池可能的结合

**Comment by xrdavies** [comment]
Rebase Community

- 吸纳新的开源项目 Web3Camp
- 14期日报
- 协办 DAO 黑山
